### DISCUSSION TITLE
[Feature] Better cover photo for person

### DISCUSSION THREAD

**Discussion by LoggeL** [immich-app/immich#6702] [Feature Request]
### The feature

The automatically chosen cover photos for each person are sometimes very low resolution. Wouldn't it be better to re-check those after a couple of images got assigned to a person and to set a minim pixel threshold? This would already increase the quality of the people page and require less manual effort.

### Platform

- [X] Server
- [x] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of the more active #10051
